src/services/featureoverlay.js

```diff
@@ -239,4 +277,24 @@ ngeo.FeatureOverlay.prototype.setStyle = function(style) {
 };
 
 
+/**
+ * @param {ol.CollectionEvent} evt Feature collection event.
+ * @private
+ */
+ngeo.FeatureOverlay.prototype.handleFeaturesAdd_ = function(evt) {
```

Comment:
It should be called `handleFeatureAdd_` since we add only one feature at a time.